site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -56,10 +56,10 @@ The tenant in which to create this Application.
 This field is only displayed once multiple tenants exist in FusionAuth. When only a single tenant exists, the Application will always be created in the default tenant.
 
 [field]#Theme# [optional]#Optional# [since]#Available since 1.27.0#::
-When a theme is selected, it will be used for this application instead of the tenant theme. 
+When a theme is selected, it will be used for this application instead of the tenant theme.
+
+A paid edition of FusionAuth is required to use the application themes.
```

Comment:
We could optionally update this to use the new "fancy" pattern for paid features.

```
+
:premium_feature: application themes
include::../shared/_premium-edition-blurb-api.adoc[]
+
:premium_feature!:
```

Reply:
Changed.